src/Android/Hanselman.Android.csproj

```diff
@@ -213,6 +213,15 @@
       <Version>4.1.0.581479</Version>
     </PackageReference>
   </ItemGroup>
+  <ItemGroup Condition="$([System.Version]::Parse($(XamarinAndroidVersion.Split('-')[0]))) &gt;= $([System.Version]::Parse('9.4.0'))">
```

Comment:
THAT IS FANCY!

Reply:
I just saw that VS2017 - and the others ignore this condition... Maybe I am doing something wrong...